ngi_pipeline/conductor/launchers.py

```diff
@@ -20,7 +20,15 @@ def get_engine_for_bp(project, config=None, config_file_path=None):
     :param NGIProject project: The project to get the engine from.
     """
     charon_session = CharonSession()
-    best_practice_analysis = charon_session.project_get(project.project_id)["best_practice_analysis"]
+    try:
+        best_practice_analysis = charon_session.project_get(project.project_id)["best_practice_analysis"]
```

Comment:
And this error never actually gets printed anywhere. And almost all my errors are so god damned informative and boring, I gotta vent somewhere.